README.md

```diff
@@ -1,11 +1,31 @@
 # STAC ipyleaflet
 
+### What
+stac_ipyleaflet is a customized version of [ipyleaflet](https://ipyleaflet.readthedocs.io/en/latest/) built to be an in jupyter-notebook interactive mapping library that can access STAC catalogs (currently only the MAAP STAC) to help better visualize cloud-optimized geotiffs. The library provides widgets that is meant to help users working in Jupyter Notebook environments to quickly visualize and interact with geospatial data at different stages throughout their process.
+
+### Why
+While the intended users of this library are technical and can write code on their own to plot and analyze data, it's very time consuming to export results and visualize/explore data in & out of a jupyter notebook. So this library helps users visualize remotely sensed data as quickly as possible at scale as fast as possible.
+
+![stac_ipyleaflet as part of the explore/visualization solution](/public/images/about-map-visualization-solution.png)
+> stac_ipyleaflet as part of the explore/visualization solution
+
 WORK IN PROGRESS. Right now this connects to the MAAP STAC - providing a module on top of ipyleaflet demonstrating how to load & control opacity for tile layers (from `biomass-layers.csv`), view pre-determined Basemaps, and derive coordinates from a user-defined bounding box.
 
 Much of this project is inspired from [leafmap](https://leafmap.org/)
 
 ![Jupyter Lab ScreenShot](/public/images/jlab-screenshot.png)
 
+## Features
+* Layers widget with Biomass and Basemap layers with opacity control
+* STAC integration to display COGs on map
+* Ability to draw AOI and copy coordinates
+
+## Contributing
+@TODO - decide and finalize on git conventions like branching standards also testing coverage
+To contribute to this codebase, clone this repository down and create a new branch. All PRs should be against the `main` branch. Branch names should be prefixed with either *feature, bugfix, docs, refactor*. 
+
+For example when adding documentation, the branch name should look something like `docs/{special_branch_name}`. Or when refactoring for code optimization, the branch name should look something like `refactor/{special_branch_name}`
+
```

Comment:
📓 : This is WIP and will need to be developed / finalized with @emmalu and anyone else